apps/comps/components/unified-leaderboard/skill-detail/skill-detail-leaderboard-table.tsx

```diff
@@ -434,6 +429,30 @@ export const SkillDetailLeaderboardTable: React.FC<
                       )}
                     </div>
                   </div>
+                </>
+              );
+
+              return (
+                <div key={participant.id} className="group">
+                  {isModel ? (
+                    <a
+                      href={`https://openrouter.ai/models/${participant.id}`}
+                      target="_blank"
+                      rel="noopener noreferrer"
+                      className="block rounded-lg transition-colors hover:bg-gray-700/50"
```

Comment:
let's add `px-2 py-1` here and below
```ts
className="block rounded-lg px-2 py-1 transition-colors hover:bg-gray-700/50"
```
so the row padding on hover has some more space

<img width="1233" height="237" alt="Image" src="https://github.com/user-attachments/assets/fcb0d7ad-6724-4525-99f6-89b00a129d87" />

Reply:
Yeah that def looks better - thanks! Added here [ba984fc](https://github.com/recallnet/js-recall/pull/1456/commits/ba984fc006b66d4a5d97b7771888cadf95519d4b)